File: DatingApp-SPA/src/app/_models/message.ts
export interface Message {
    id: number;
    senderId: number;
    senderNickName: string;
    senderPhotoUrl: string;
    recipientId: number;
    recipientNickName: string;
    recipientPhotoUrl: string;
    content: string;
    isRead: boolean;
    dateRead: Date;
    messageSent: Date;

}

File: DatingApp.API/Models/Message.cs
using System;

namespace DatingApp.API.Models
{
    public class Message
    {
        public int id { get; set; }
        public int senderId { get; set; }
        public User sender { get; set; }
        public int recipientId { get; set; }
        public User recipient { get; set; }
        public string content { get; set; }
        public bool isRead { get; set; }
        public DateTime? dateRead { get; set; }
        public DateTime? messageSent { get; set; }
        public bool senderDeleted { get; set; }
        public bool recipientDeleted { get; set; }

    }
}